src/node_contextify.cc

```diff
@@ -409,10 +410,15 @@ class ContextifyContext {
     Local<Object> proxy_global = PersistentToLocal(isolate,
                                                    ctx->proxy_global_);
 
-    bool in_sandbox = sandbox->GetRealNamedProperty(property).IsEmpty();
-    bool in_proxy_global =
-        proxy_global->GetRealNamedProperty(property).IsEmpty();
-    if (!in_sandbox || !in_proxy_global) {
+    if (sandbox->HasRealNamedProperty(property)) {
+      PropertyAttribute propAttr =
+        sandbox->GetRealNamedPropertyAttributes(property).FromJust();
```

Comment:
The .FromJust() here feels bad. I am not sure what the right way to handle these new maybes is.

Maybe if they're empty I should return None?